### PR TITLE
Packages with rest-xml protocol.

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: generated/aws_autoscaling_api/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.0.1
+
+(git hash: 9f95ffeaee28d3c62ee0380c32b58f27491840fe)
+
+- initial release
+
```
Comment (Schwusch) [inline generated/aws_autoscaling_api/CHANGELOG.md]: What is the point with these additions?

File: generator/config.yaml [hunk 2]
```diff
@@ -36,3 +36,7 @@ packages:
   - aws_sns_api
   - aws_sqs_api
   - aws_sts_api
+## rest-xml protocol APIs
+  - aws_cloudfront_api
+  - aws_s3_api
+  - aws_s3control_api
```
Comment (Schwusch) [inline generator/config.yaml]: Were these not generated thanks to this addition?

File: generated/aws_autoscaling_api/CHANGELOG.md (hunk 1 above)
Comment (isoos) [reply generated/aws_autoscaling_api/CHANGELOG.md]: Follows the same pattern that we will be using in the later releases.

**Comment by isoos** [comment]
Ah, wait, something is wrong with this PR, I did something wrong...

**Pull request closed without merging**